wrappers/javascript/aries-askar-nodejs/tests/migration.test.ts

```diff
@@ -0,0 +1,51 @@
+import { ariesAskar, Migration } from '@hyperledger/aries-askar-shared'
+import fs from 'fs'
+import path from 'path'
+
+const DB_TEMPLATE_PATH = path.join(__dirname, 'indy_wallet_sqlite.db')
+const DB_UPGRADE_PATH = path.join(__dirname, 'indy_wallet_sqlite_upgraded.db')
+
+describe('migration', () => {
+  beforeAll(() => {
+    require('@hyperledger/aries-askar-nodejs')
+  })
+
+  beforeEach(() => {
+    const tplPaths = [DB_TEMPLATE_PATH, `${DB_TEMPLATE_PATH}-shm`, `${DB_TEMPLATE_PATH}-wal`]
```

Comment:
Why do we also copy the other files `shm` and `wal`? Are these needed?

Reply:
I copied this setup code from the `tests/migration.rs`. I am not to well-versed on sql stuff, but it seemed that they were sometimes created and just to be safe I check here as well.

Reply:
These files are created because using the WAL mode. Usually they are 0-byte long or deleted when DB is gracefully closed. If not, they are used to process the uncommitted transactions upon the next time the DB is opened. So copying them might be useful in case you are not 100% sure that the DB was previously closed properly.

Reply:
But wouldn't that cause invalid transactions to be comitted as we do changes to the database? Or those would run before any other queries?